src/main/java/net/dries007/tfc/command/CommandFindVeins.java

```diff
@@ -99,7 +221,110 @@ public int getRequiredPermissionLevel()
         {
             return getListOfStringsMatchingLastWord(args, VeinRegistry.INSTANCE.keySet());
         }
+        else if (args.length == 3)
+        {
+            return getListOfStringsMatchingLastWord(args, "dump", "rate");
+        }
         return Collections.emptyList();
     }
+
+    private static class Worker implements WorldWorkerManager.IWorker
+    {
+        private final int dimension;
```

Comment:
Sure thing. (was just a copy-paste from forge, and since i wasn't too sure if we would ever generate veins in other dimensions past 1.15, I left there)